### ISSUE TITLE
Running any Report on the Server fails

### ISSUE THREAD

**Issue by mathewlsm** [bitnami/bitnami-docker-jasperreports#58]
**Description**

Running any Report on the Server fails. A Server Error with an UID appears. Database Connection etc. is tested and works correctly. The Report works when running local.

**Steps to reproduce the issue:**

1. Adding a new Report to the repository with a local working JRXML-File and a testet Datasource.
2. Opening the report from the library

**Describe the results you received:**

An error appears with a UID. The Log shows

`
jasperreports_1  | 2019-01-07 11:56:42,848 ERROR ErrorPageHandlerAction,http-nio-8080-exec-3:115 - Error UID ca46e260-f0da-4e01-9c48-21749f583bcb
jasperreports_1  | com.jaspersoft.jasperserver.api.JSException: java.lang.UnsatisfiedLinkError: /opt/bitnami/java/lib/amd64/libfontmanager.so: libfreetype.so.6: cannot open shared object file: No such file or directory
jasperreports_1  | Arguments:
jasperreports_1  |      at com.jaspersoft.jasperserver.api.engine.jasperreports.service.impl.EngineServiceImpl.fillReport(EngineServiceImpl.java:672)
jasperreports_1  |      at com.jaspersoft.jasperserver.api.engine.jasperreports.service.impl.EngineServiceImpl.executeReportUnitRequest(EngineServiceImpl.java:2112)
jasperreports_1  |      at com.jaspersoft.jasperserver.api.engine.jasperreports.domain.impl.ReportUnitRequest.execute(ReportUnitRequest.java:64)
jasperreports_1  |      at com.jaspersoft.jasperserver.api.engine.jasperreports.service.impl.EngineServiceImpl.execute(EngineServiceImpl.java:541)
jasperreports_1  |      at com.jaspersoft.jasperserver.war.action.ViewReportAction.executeReport(ViewReportAction.java:672)
jasperreports_1  |      at com.jaspersoft.jasperserver.war.action.ViewReportAction.executeReport(ViewReportAction.java:557)
jasperreports_1  |      at com.jaspersoft.jasperserver.war.action.ViewReportAction.runReport(ViewReportAction.java:433)
`

**Describe the results you expected:**

We expect that the report will open correctly.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Version**

- Output of `docker version`:

```
Client:
 Version:      17.03.2-ce
 API version:  1.27
 Go version:   go1.6.2
 Git commit:   f5ec1e2
 Built:        Thu Jul  5 23:07:48 2018
 OS/Arch:      linux/amd64

Server:
 Version:      17.03.2-ce
 API version:  1.27 (minimum version 1.12)
 Go version:   go1.6.2
 Git commit:   f5ec1e2
 Built:        Thu Jul  5 23:07:48 2018
 OS/Arch:      linux/amd64
 Experimental: false
```

- Output of `docker info`:

```
Containers: 35
 Running: 26
 Paused: 0
 Stopped: 9
Images: 26
Server Version: 17.03.2-ce
Storage Driver: aufs
 Root Dir: /data/docker/aufs
 Backing Filesystem: extfs
 Dirs: 263
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 4ab9917febca54791c5f071a9d1f404867857fcc
runc version: 54296cf40ad8143b62dbcaa1d90e520a2136ddfe
init version: N/A (expected: 949e6facb77383876aeff8a6944dde66b3089574)
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 4.4.0-138-generic
Operating System: Ubuntu 16.04.5 LTS
OSType: linux
Architecture: x86_64
CPUs: 8
Total Memory: 23.55 GiB
Name: gisu031
ID: BX2V:UNQK:WX4E:37IH:INAT:6BBQ:XOQD:FAWZ:YJJR:SXBJ:WVRM:KJ3Z
Docker Root Dir: /data/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false

WARNING: No swap limit support
```

- Output of `docker-compose version` (if applicable):

```
docker-compose version 1.8.0, build unknown
docker-py version: 1.9.0
CPython version: 2.7.12
OpenSSL version: OpenSSL 1.0.2g  1 Mar 2016
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**

**Comment by javsalgar** [comment]
Hi,

We are releasing a new revision of the container that includes freetype. We will let you know as soon as it is out.

**Comment by carrodher** [comment]
Hi! Sorry for not coming back with an answer, a version with "freetype" was released some weeks ago.

**Comment by marisamartinserrano** [comment]
Hi!


I still experience the error described above when I open an existing or a new Report via the web console.
I deployed the following version using helm stable/jasperreports chart:
- registry: docker.io
- repository: bitnami/jasperreports
- tag: 7.2.0-debian-10-r0

I followed instructions on https://community.jaspersoft.com/questions/1084096/change-sql-security-validation-settings updating the files below and restarting the container:
- opt/bitnami/jasperreports/WEB-INF/classes/esapi/validation.properties
- opt/bitnami/jasperreports/WEB-INF/classes/esapi/security-config.properties

Thank you!

**Jasper Reports web console:**
Error Message
There was an error on the server. Try again or contact site administrators. (Error UID: d68e315c-c7fe-4405-be47-aad137fc903c)


**Jasper Reports pod/container logs:**
2020-01-26 19:47:46,377 ERROR SecureExceptionHandlerImpl,http-nio-8080-exec-14:116 - There was an error on the server. Try again or contact site administrators. (Error UID: d68e315c-c7fe-4405-be47-aad137fc903c)
--
  | 2020-01-26 19:47:46,378 ERROR ErrorPageHandlerAction,http-nio-8080-exec-14:118 - Error UID d68e315c-c7fe-4405-be47-aad137fc903c
  | java.util.MissingResourceException: Can't find bundle for base name tr, locale en
